media/css/custom.scss

```diff
@@ -0,0 +1,471 @@
+  height: 25px;
+  padding: 6px;
+  text-decoration: none;
+}
+
+.white {
+  color: $white;
+}
+
+.article {
+  .thumbnail {
+    margin: 15px;
+  }
+}
+
+a {
+  color: $sidebar-background;
+  &:hover {
+    color: $sidebar-link-highlight;
+  }
+  &:focus {
+    outline: thin dotted $sidebar-link-highlight;
+    outline-offset: -2px;
+  }
+}
+
+.well {
+  background-color: $sidebar-background;
+  li a {
+    color: $white;
+  }
+}
+
+a.thumbnail {
+  background-color: $white;
+  &:hover {
+    background-color: $sidebar-link-highlight;
+  }
+}
+
+#sidebar {
+  .well,
+  h2,
+  a {
+    color: $white;
+  }
+  a:hover {
+    color: $sidebar-link-highlight;
+  }
+}
+
+#panel1,
+#panel2,
+#panel3 {
+  background: $sidebar-background;
+  color: $white;
+  display: none;
+  height: 40px;
+  padding: 5px;
+  width: 225px;
+}
+
+#photo-feed {
+  > .span3 {
+    height: 350px;
+    margin: 5px 5px 25px;
+  }
+  .thumbnail {
+    height: inherit;
+    & > img {
+      height: 200px;
+      width: 210px;
+    }
+    .caption {
+      h4 {
+        height: 3em;
+        overflow: hidden;
+      }
+    }
+  }
+  .caption {
+    p {
+      font-size: 14px;
+    }
+  }
+}
+
+.twitter-follow-button {
+  margin-top: 3px;
+}
+
+.btn-danger {
+  background: $sidebar-link-highlight;
+  &:hover {
+    background: $danger-link-highlight;
+  }
+}
+
+h1,
+h2,
+h3 {
+  color: $dark-blue;
+}
+
+.leftimg,
+.deanimg {
+  height: 300px;
+}
+
+.leftimg {
+  float: left;
+  margin: 10px 10px 20px 0;
+}
+
+.deanimg {
+  margin: 20px 100px 20px 0;
+}
+
+.codehilite {
+  clear: left;
+}
+
+.badge {
+  font-size: 11px;
+  padding-bottom: 3px;
+  padding-left: 6px;
+}
+
+.badge-important {
+  background-color: $sidebar-link-highlight;
+}
+
+#loader {
+  background-color: $sidebar-link-highlight;
+  margin-bottom: 20px;
+  padding-bottom: 3px;
+}
+
+#spinner {
+  display: inline;
+  margin-right: 3px;
+}
+
+*::selection,
+*::-moz-selection,
+*::-webkit-selection {
+  background: $bright-red;
+  color: $white;
+}
+
+@import "social_buttons";
+@import "navigation";
+
+* {
+  // old-style reset here
+  border: 0;
+  padding: 0;
+}
+
+table {
+  border: 1px solid $grayish-blue;
+  border-collapse: separate;
+  border-width: 0 0 1px 1px;
+  font-size: 20px;
+}
+
+th {
+  font-size: 14px;
+  font-weight: bold;
+  position: relative;
+  text-align: center;
+  vertical-align: middle;
+}
+
+td:hover,
+th:hover {
+  background-position: 0 -81px;
+  color: $light-black;
+}
+
+.events {
+  position: relative;
+  ul {
+    background: $light-grayish-blue;
+    border: 1px solid $white;
+    @include border-radius(3px);
+    -webkit-box-shadow: 0 8px 8px $dark-gray;
+    box-shadow: 0 8px 8px $dark-gray;
+    color: $gray;
+    display: none;
+    font-size: 15px;
+    list-style: none;
+    padding: 15px;
+    position: absolute;
+    text-align: left;
+    width: 600px; // 200px, wow banner
+    z-index: 1000;
+  }
+  li {
+    padding-bottom: 5px;
+    span {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3